ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -63,7 +98,7 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
     const { isConnecting } = transition.to.queryParams;
 
     if (isConnecting) {
-      if (model.target.address || model.hosts.length === 1) {
+      if (model.target.address || model.hosts.length <= 1) {
```

Comment:
Do we need the `model.target.address` if we're checking for `model.hosts.length <= 1`? Seems like that covers the address case now

Reply:
@ZedLi 
That is correct, with the new condition the `model.target.address` is already covered.